janus.c

```diff
@@ -3873,8 +3873,10 @@ gint main(int argc, char *argv[])
 	JANUS_LOG(LOG_INFO, "Destroying sessions...\n");
 	if(sessions != NULL)
 		g_hash_table_destroy(sessions);
+	sessions = NULL;
 	if(old_sessions != NULL)
 		g_hash_table_destroy(old_sessions);
+	old_sessions = NULL;
```

Comment:
Yes, that does look cleaner.  The same pattern could be used for [this](https://github.com/MotorolaSolutions/janus-gateway/blob/1b9e4f0b164bb986c00463db3acabcef1d532928/janus.c#L3293):
`g_strfreev(list);`
`list = NULL;`
change to:
`g_clear_pointer(&list, g_strfreev);`